aws/resource_aws_subnet_test.go

```diff
@@ -438,13 +484,52 @@ resource "aws_vpc" "test" {
 resource "aws_subnet" "test" {
   cidr_block              = "10.1.1.0/24"
   vpc_id                  = aws_vpc.test.id
-  map_public_ip_on_launch = true
+	map_public_ip_on_launch = true
```

Comment:
Nit: Whitespace adjustment

```suggestion
  map_public_ip_on_launch = true
```